Lib/test/test_pdb.py

```diff
@@ -1110,6 +1112,44 @@ def test_readrc_kwarg(self):
             if save_home is not None:
                 os.environ['HOME'] = save_home
 
+    def test_find_function_with_invalid_filename(self):
+        self.assertIsNone(pdb.find_function('foo', 'invalid filename'))
+
+    def test_getsourcelines(self):
+        def foo():
+            pass  # pragma: no cover
+        lines, lineno = inspect.findsource(foo)
+        expected_lines = inspect.getblock(lines[lineno:])
+        expected_lineno = lineno + 1
+        actual_lines, actual_lineno = pdb.getsourcelines(foo)
+        self.assertEqual(actual_lines, expected_lines)
+        self.assertEqual(actual_lineno, expected_lineno)
+
+    def test_getsourcelines_with_module_frame_obj(self):
+        for frame, *unused in inspect.getouterframes(inspect.currentframe()):
```

Comment:
Ah, I'm sorry.  I misread "I need" as "I don't need".

Now I read pdb.getsourcelines().
If I'm correct, do you want to test this one line: `return lines, 1`?

I don't know way to create module frame which is simple enough to test such simple one line.